crates/uniffi-bindgen-react-native/src/bindings/react_native/gen_cpp/filters.rs

```diff
@@ -15,25 +16,24 @@ pub fn ffi_type_name_to_rust(ffi_type: &FfiType) -> Result<String, askama::Error
 
 fn ffi_type_name(ffi_type: &FfiType) -> Result<String, askama::Error> {
     Ok(match ffi_type {
-        FfiType::UInt8 => "uint8_t",
-        FfiType::Int8 => "int8_t",
-        FfiType::UInt16 => "uint16_t",
-        FfiType::Int16 => "int16_t",
-        FfiType::UInt32 => "uint32_t",
-        FfiType::Int32 => "int32_t",
-        FfiType::UInt64 => "uint64_t",
-        FfiType::Int64 => "int64_t",
-        FfiType::Float32 => "float",
-        FfiType::Float64 => "double",
-        FfiType::RustArcPtr(_) => "void *",
-        FfiType::RustBuffer(_) => "RustBuffer",
-        FfiType::ForeignBytes => "ForeignBytes",
-        FfiType::Callback(_) => "Callback",
-        FfiType::Struct(_) => todo!(),
-        FfiType::Handle => "Handle",
+        FfiType::UInt8 => "uint8_t".into(),
+        FfiType::Int8 => "int8_t".into(),
+        FfiType::UInt16 => "uint16_t".into(),
+        FfiType::Int16 => "int16_t".into(),
+        FfiType::UInt32 => "uint32_t".into(),
+        FfiType::Int32 => "int32_t".into(),
+        FfiType::UInt64 => "uint64_t".into(),
+        FfiType::Int64 => "int64_t".into(),
+        FfiType::Float32 => "float".into(),
+        FfiType::Float64 => "double".into(),
+        FfiType::RustArcPtr(_) => "void *".into(),
+        FfiType::RustBuffer(_) => "RustBuffer".into(),
+        FfiType::ForeignBytes => "ForeignBytes".into(),
+        FfiType::Callback(_) => "Callback".into(),
+        FfiType::Struct(nm) => format!("Uniffi{}", nm.to_upper_camel_case()), // XXX
```

Comment:
Just for my own knowledge, what is the // XXX here, just a marker to revisit?

Reply:
Yes, precisely so. My comment [below](https://github.com/jhugman/uniffi-bindgen-react-native/pull/4/files/4925eb51c8c791bed7cd9de39aa7c38dd093671e#r1610368990) may or may not be useful.